app/logic/EditionsChecker.scala

```diff
@@ -0,0 +1,52 @@
+package logic
+
+import model.editions.{EditionsCollection, EditionsFront, EditionsIssue}
+
+object EditionsChecker {
+
+  def check(issue: EditionsIssue): List[String] =
+    if (issue.fronts.isEmpty)
```

Comment:
so only the first function is public the rest is private,
can we make the other `check` function private it will be easier to read

Reply:
Done, plus some other refactoring for clarity.